public-ui/src/app/views/survey-view/survey-view.component.ts

```diff
@@ -78,6 +78,37 @@ export class SurveyViewComponent implements OnInit, OnDestroy {
           let didNotAnswerCount  = this.survey.participantCount;
           q.selectedAnalysis = q.genderAnalysis;
           for (const a of q.countAnalysis.results) {
+            if (q.subQuestions) {
+              let matchedQuestionConcepts: QuestionConcept[] = [];
+              matchedQuestionConcepts = q.subQuestions.filter(
+                w => w.conceptName.toLowerCase() === a.stratum4.toLowerCase());
+              if (matchedQuestionConcepts.length === 0) {
+                const missingQuestionConcept = {
```

Comment:
nit: I think assigning to this variable is making it less readable, would just inline it below

Reply:
Done.Thanks